.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,88 @@
+name: Test Emscripten/Pyodide build
+# TODO: move the relevant workflow items to Azure Pipelines
+# when the build and tests pass on GitHub Actions here
+# and then remove this workflow file altogether.
+
```

Comment:
Though I've marked this comment here: I have also found that GitHub Actions is ~3-4 minutes faster in comparison to Azure Pipelines – I'll do what the maintainers suggest and move the workflow to whichever CI provider what is requested of me :D